plantisc2/tasks/app.yml

```diff
@@ -6,3 +6,14 @@
     version: remove-archive
   become: yes
   become_user: www-data
+
+- name: Copy sql Script
+  template:
+    src: tissue.sql
```

Comment:
Actually `tissue.sql` is the script tha creates the tissue database, I don't know if templates is the right place where to put it. if it is I'll change its name accordingly.